docs/data/charts/gauge/gauge.md

```diff
@@ -103,10 +103,10 @@ import {
 
 To create your own components, use the `useGaugeState` hook which provides all you need about the gauge configuration:
 
-- information about the value: `value`, `valueMin`, `valueMax`;
-- information to plot the arc: `startAngle`, `endAngle`, `outerRadius`, `innerRadius`, `cornerRadius`, `cx`, and `cy`;
+- information about the value: `value`, `valueMin`, `valueMax`
+- information to plot the arc: `startAngle`, `endAngle`, `outerRadius`, `innerRadius`, `cornerRadius`, `cx`, and `cy`
 - computed values:
-  - `maxRadius` the maximal radius that can fit the drawing area;
+  - `maxRadius` the maximal radius that can fit the drawing area
   - `valueAngle` the angle associated with the current value.
```

Comment:
Those semi colon come from the following link Lukas pointed.

https://www.imperial.ac.uk/brand-style-guide/writing/grammar/punctuation/bullet-points/#:~:text=For%20a%20list%20of%20longer,work%20with%20the%20introductory%20sentence.

@samuelsycamore do you have an opinion on this topic?